server/controllers/books.js

```diff
@@ -0,0 +1,346 @@
+import { Book, BorrowedBook, BookCategory } from '../models';
```

Comment:
Check that the return tag in your JSDocs for your methods state clearly the right data being returned.